pages/test.tsx

```diff
@@ -0,0 +1,17 @@
+import Sidebar from 'components/Sidebar/Sidebar'
```

Comment:
once u rename the sidebar file u can just do `import Sidebar from 'components/Sidebar`